site/source/downloads.html.erb

```diff
@@ -36,18 +36,28 @@ breadcrumb: Downloads
       </ul>
       </p>
 
-      <p>Once you have downloaded Mesos, follow the
+      <p>
+        Once you have downloaded Mesos, follow the
         <a href="/getting-started/">getting started instructions</a>,
         and take a look at the rest of the
         <a href="/documentation/">Mesos documentation</a>.
       </p>
 
       <h4>Getting the code via source control</h4>
-      <p>The codebase can also be downloaded from the Apache git repository at:
+      <p>
+        The codebase can also be downloaded from the Apache git repository at:
         <code>https://gitbox.apache.org/repos/asf/mesos.git</code>.
         <br />
         This repository is also mirrored on
         <a href="https://github.com/apache/mesos">GitHub</a>.
       </p>
+
+      <h4>Getting community driven Mesos binaries</h4>
+      <p>
+        <i>The following repositories are unofficial and not supportet by Apache.</i>
+        <ul>
+          <li>Mesos binaries for CentOS (x86), AlmaLinux (x86, aarch64), Ubuntu (x86): <a href="https://rpm.aventer.biz">https://rpm.aventer.biz</a></li>
```

Comment:
I think it is `x86_64` rather than `x86`, right?

And I'd suggest to add a newline after `<li>`.